pkg/virtctl/usbredir/usbredir.go

```diff
@@ -141,6 +141,10 @@ func (usbredirCmd *usbredirCommand) Run(command *cobra.Command, args []string) e
 		select {
 		case <-usbredirDoneChan: // Wait for local usbredir to complete
 		case err = <-streamStop: // Wait for remote connection to close
+			if err == nil {
```

Comment:
Wouldn't this yield EOF anyway?

Reply:
It doesn't if the connection is closed by server side

Reply:
Thanks for explanation!